views/inputs/integer.php

```diff
@@ -12,7 +12,10 @@
  */
 $input = $this->input;
 
-$input['type']  = 'text';
-$input['class'] = sprintf( '%s %s', $input['class'] ?? '', 'gla-input-integer' );
+$input['type'] = 'number';
+// Not so "custom" but standard `<input type="numer">` attribute.
```

Comment:
Just a tiny typo here.

```suggestion
// Not so "custom" but standard `<input type="number">` attribute.
```